thirdparty/haw_logger/logger/log_base.h

```diff
@@ -42,7 +42,7 @@
 #define DEPRECATED LOGD() << "This function deprecated!!"
 #define DEPRECATED_USE(use) LOGD() << "This function deprecated!! Use:" << use
 #define NOT_IMPLEMENTED LOGW() << "Not implemented!!"
-#define NOT_IMPL_RETURN NOT_IMPLEMENTED return
+#define NOT_IMPL_RETURN NOT_IMPLEMENTED; return
```

Comment:
Why?

Reply:
Otherwise it would expand to `LOGW() << "Not implemented!!" return`, wouldn't it? That is not valid.

Reply:
Oops, of course. I misses that it is 3 'words'